grails-app/views/layouts/main_component.gsp

```diff
@@ -194,15 +194,23 @@
 <g:render template="/base/topNav" />
 
 <auth:isNotAuthenticated>
-    <div class="container">
-        <div id="login_spinner" class="hidden">
-            <div class="alert alert-success alert-dismissable" style="display: block">
-                Loading ... <span class="glyphicon glyphicon-refresh glyphicon-refresh-animate"></span>
-            </div>
-        </div>
-        <h3>About the ORSP Portal</h3>
-        <g:render template="/index/aboutBlurb"/>
-    </div>
+  <g:if test="${pageProperty(name:'body.dul') == 'true'}">
```

Comment:
This implementation is an anti-pattern. If we generalize this approach, then we have custom if blocks that are dependent on strings in unrelated gsps spread throughout the application - that is a brittle and un-refactor-able approach. There should be as close to zero custom logic in `main_component.gsp` as possible - `dataUseLetter/index.gsp` should be doing all of the work necessary for determining what kind of layout it is enclosed within. Instead, you should have a customizable `<meta name="layout" content="main_component"/>` in `dataUseLetter/index.gsp` that is smart enough to choose a parent layout that does not include the unwanted content.